src/v/config/configuration.cc

```diff
@@ -3697,6 +3697,23 @@ configuration::configuration()
       {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
       std::chrono::milliseconds(1min),
       {.min = 10ms})
+  , iceberg_catalog_commit_interval_ms(
+      *this,
+      "iceberg_catalog_commit_interval_ms",
+      "The frequency at which the Iceberg coordinator commits topic files to "
+      "the catalog. This is the interval between commit transactions across "
+      "all topics monitored by the coordinator, not the interval between "
+      "individual commits.",
+      {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
+      std::chrono::milliseconds(1min),
+      {.min = 10ms})
+  , iceberg_catalog_base_location(
+      *this,
+      "iceberg_catalog_base_location",
+      "Base path for the cloud object storage-backed Iceberg catalog. After "
+      "Iceberg is enabled, do not change this value.",
```

Comment:
>      "Iceberg is enabled, do not change this value.",

do you think that redpanda should make it a bit harder to change this? one way might be to create a custom property type that flipped the visibility to `visibility::tunable` once a particular set of conditions were met.

Reply:
>custom property type that flipped the visibility

that's a cool idea. a bit heavy handed, but isn't it possible to do like multiproperty validations? i.e. reject any change to `base_location` if `enabled` is true?

Reply:
yeh lots of options.